add clarification on remote vs remote state backends (#1851)
opentofu/opentofu: website/docs/language/settings/backends/remote.mdx

# Backend Type: remote

Most of the available backends provide different ways of storing state snapshots remotely. Therefore, they are often called remote state backends.

> This page describes a special `remote` backend, that has nothing to do with remote state backends.

Reviewer: ```suggestion
This page describes a special `remote` backend, not to be confused with the simpler remote state backends.
```
